python/src/mas/cli/install/catalogs.py

```diff
@@ -19,5 +19,9 @@
         "v9-250306-s390x",
         "v9-250206-s390x",
         "v9-250109-s390x",
+        "v9-241205-s390x",
```

Comment:
was this old catalog missed before? why is it being added now?

Reply:
This is removed , file is outdated